Sparkle/SUFileManager.m

```diff
@@ -178,9 +178,9 @@ - (BOOL)_removeXAttrWithAuthentication:(NSString *)name fromRootURL:(NSURL *)roo
         }
         return NO;
     }
-    
-    const char *xattrName = [name cStringUsingEncoding:NSASCIIStringEncoding];
-    if (xattrName == NULL) {
+
+    char xattrName[PATH_MAX] = {0};
+    if (![name getFileSystemRepresentation:xattrName maxLength:sizeof(xattrName)]) {
```

Comment:
Why? `xattrName` isn't a path.

It's merely the `"com.apple.quarantine"` constant. Maybe instead of the conversion just use it as `const char *` everywhere?